python_functions/test/scalar/sample_test.py

```diff
@@ -0,0 +1,6 @@
+from python_functions.udfs.scalar.sample import sample
```

Comment:
lets have a prefix `dagger-` in the package name. That way it would be consistent with other packages